### ISSUE TITLE
Route popularity indicator not working correctly with few ascentes

### ISSUE THREAD

**Issue by killakalle** [theCrag/website#3856]
**What happened?**
A route with two ascents gets full popularity score. A route with 1 ascent gets 0 popularity score - same as a route with 0 ascents.

Example url(s) to reproduce the problem:
https://www.thecrag.com/es/escalar/spain/valencia/area/4371832137

Another example with routes with 1 ascent not distinguished from routes with 0 ascents
https://www.thecrag.com/es/escalar/spain/valencia/area/3927257532

Ideally also provide screenshots:
![image](https://user-images.githubusercontent.com/30682796/108608591-4ac4d880-73c8-11eb-924d-a8dca2801013.png)

**What you expected:**

* [ ] On Sectors with low total number of ascents Routes with 1 ascent show a different popularity rating than 0 ascent routes

**Comment by DaneEvans** [comment]
I disagree with this one. IMO It really shouldn't get anything in the bar
until the area has at least an average of 2 ascents per climb.
Otherwise it's a completely arbitrary number
